### PR TITLE
Add AIO doc reference to npm package readme

### PR DESCRIPTION
**Pull request by jbogarthyde**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
 Current npm doc for Angular packages is a generic README that refers to sources in git, but no information on what the package does. 
For example: https://www.npmjs.com/package/@angular/router

Issue Number:  https://github.com/angular/angular/issues/33766

## What is the new behavior?
Adding a ref to AIO for usage/reference info on packages.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by gkalpak — APPROVED** [review]

**Comment by gkalpak** [comment]
@jbogarthyde, you need to update the golden files as well (to make CI green). Here is the [CI error][1]:

```
Failures:
1) Comparing test packages to golds Package "Example NPM package"
  Message:
    Failed: example ng_package differs from golden file
        Diff:
        --- /b/f/w/bazel-out/k8-fastbuild/bin/packages/bazel/test/ng_package/example_package.sh.runfiles/angular/packages/bazel/test/ng_package/example_package.golden	Golden file
    +++ /b/f/w/bazel-out/k8-fastbuild/bin/packages/bazel/test/ng_package/example_package.sh.runfiles/angular/packages/bazel/test/ng_package/example_package.golden	Generated file
    @@ -56,10 +56,12 @@
     Angular
     =======
     
     The sources for this package are in the main [Angular](https://github.com/angular/angular) repo. Please file issues and pull requests against that repo.
     
    +Usage information and reference details can be found in [Angular documentation](https://angular.io/docs).
    +
     License: MIT

     
     --- arbitrary-npm-package-main.js ---
    
        To accept the new golden file, run:
          yarn bazel run //packages/bazel/test/ng_package:example_package.accept

...

2) Comparing test packages to golds Package "Example with ts_library NPM package"
  Message:
    Failed: example ng_package differs from golden file
        Diff:
        --- /b/f/w/bazel-out/k8-fastbuild/bin/packages/bazel/test/ng_package/example_package.sh.runfiles/angular/packages/bazel/test/ng_package/example_with_ts_library_package.golden	Golden file
    +++ /b/f/w/bazel-out/k8-fastbuild/bin/packages/bazel/test/ng_package/example_package.sh.runfiles/angular/packages/bazel/test/ng_package/example_with_ts_library_package.golden	Generated file
    @@ -67,10 +67,12 @@
     Angular
     =======
     
     The sources for this package are in the main [Angular](https://github.com/angular/angular) repo. Please file issues and pull requests against that repo.
     
    +Usage information and reference details can be found in [Angular documentation](https://angular.io/docs).
    +
     License: MIT
     
     --- bundles/example-with-ts-library-portal.umd.js ---
    
        To accept the new golden file, run:
          yarn bazel run //packages/bazel/test/ng_package:example_package.accept
```

You either need to run `yarn bazel run //packages/bazel/test/ng_package:example_package.accept` from the root of the repo or mnually update the golden files as shown in the errors above.

[1]: https://circleci.com/gh/angular/angular/532594?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by jbogarthyde** [comment]
@gkalpak I added the new text to the golden-file tests, but I'm still getting the same error. I reinstalled node and did a new `yarn install`, but still can't run the `bazel` script: 
```
 yarn bazel run //packages/bazel/test/ng_package:example_package.accept
yarn run v1.19.1
error Command "bazel" not found.
---
bazel run //packages/bazel/test/ng_package:example_package.accept

Error: Error: corrupt installation: file 'C:\Users\jbogart/_bazel_jbogart/install/4a9f5e558d7c9ad594b85bb018406667/_embedded_binaries/A-server.jar' modified.  Please remove 'C:\Users\jbogart/_bazel_jbogart/install/4a9f5e558d7c9ad594b85bb018406667' and try again.
```

**Comment by jbogarthyde** [comment]
@gkalpak Succeeded in running script, but it failed:
```ERROR: Unrecognized option: --experimental_allow_incremental_repository_updates```

File: packages/bazel/test/ng_package/example_package.golden [hunk 1]
```diff
@@ -1110,4 +1112,3 @@ export * from './secondary/secondary';
 --- some-file.txt ---
 
 This file is just copied into the package.
-
```
Comment (gkalpak) [inline packages/bazel/test/ng_package/example_package.golden]: You should not remove these lines, because CI will be unhappy.

**Comment by gkalpak** [comment]
@googlebot I consent.

**Comment by jbogarthyde** [comment]
Thanks @gkalpak 😄

**Pull request closed without merging**